Added new config options

paigehargrave opened the pull request.

Commit:
Added new config options

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
 |Value |Description
-| *Max connections*
+| *Max Connections*
 | The maximum amount of simultaneous connections to allow.
```
amount > number

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
-| *Max requests per connection*
+| *Max Requests per Connection*
 | The maximum number of requests per connections to allow. +
 This value will determine how much reuse a connection can allow; consider the amount of CPU required to terminate and re-establish a TLS encrypted connection when lowering this value.
```
present tense:
```suggestion
Because this value determines how much reuse a connection allows, consider the amount of CPU required to terminate and reestablish a TLS-encrypted connection when lowering this value.
```

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
 
-| *Connection idle time-out*
+| *Connection Idle Time-out*
 | The maximum amount of time that you allow an idle connection. +
```
Does "you" allow the idle connection?
```suggestion
| The maximum amount of time that a connection can remain idle. +
```

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
-| *Connection idle time-out*
+| *Connection Idle Time-out*
 | The maximum amount of time that you allow an idle connection. +
 This value helps you terminate idle connections and free up resources. +
```
free up > free

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
+| *Connection Idle Time-out*
 | The maximum amount of time that you allow an idle connection. +
 This value helps you terminate idle connections and free up resources. +
 This value should always be higher than your *read request time-out*.
```
If quoting the UI label, then capitalize it as there:
```suggestion
This value should always be higher than your *Read Request Time-out*.
```
OR
```suggestion
This value should always be higher than your read request timeout.
```

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
 
-If a Mule application takes longer to respond than this value, the connection will automatically be closed. +
-This value should always be lower than the *connection idle time-out* value configured above.
+If a Mule application takes longer to respond than this value, the connection is automatically closed. +
```
```suggestion
If a Mule application takes longer than this value to respond, the connection is automatically closed. +
```

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
-If a Mule application takes longer to respond than this value, the connection will automatically be closed. +
-This value should always be lower than the *connection idle time-out* value configured above.
+If a Mule application takes longer to respond than this value, the connection is automatically closed. +
+This value should always be lower than the *connection idle time-out* value previously configured.
```
ditto re. *Connection Idle Time-out" or "connection idle timeout."

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
-| *Max pipeline depth*
+| *Read Response Time-out*
+| The maximum amount of time spent to initiate a response before the connection is terminated.+
+This value enables large payloads be terminated to keep resources available.+
```
```suggestion
This value enables requests with large payloads to be terminated to keep resources available.+
```

kbostrom replied:
GLOBALLY try to reword these texts to sound like the one in lines 238-9.

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
+*Default value*: 300 seconds
+
+| *Write Response Time-out*
+| The maximum amount of time spent from the end of the request header read to the end of the response write before the request is terminated.+
```
Can we make this more simple: "from the end of the request header read to the end of the response write"?

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
+*Default value*: 10 seconds
+
+| *Max Pipeline Depth*
 | The maximum amount of requests to allow from the same client. +
```
number

kbostrom commented on modules/ROOT/pages/enable-inbound-traffic.adoc:
```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
+| *Max Pipeline Depth*
 | The maximum amount of requests to allow from the same client. +
 This value defines how many simultaneous requests a client can send. +
 If a client exceeds this number, the exceeding requests will not be read until the requests in the queue receive a response.
```
present tense

kbostrom reviewed (approved):
Here you go!

sundippatel committed:
Update enable-inbound-traffic.adoc

Removed a clause which is no longer true, thanks to a feature we released last month.

sundippatel reviewed (dismissed):
Removed a clause which is no longer true; rest LGTM

Commit:
Style updates
